docs/sql/functions/char.md

```diff
@@ -221,16 +220,16 @@ SELECT
 
 <div class="nostroke_table"></div>
 
-| **Description** | Concatenate strings together separated by the specified separator. |
-| **Example** | `concat_ws(', ', 'Banana', 'Apple', 'Melon')` |
+| **Description** | Concatenate many strings, separated by separator. NULLs inputs are skipped. |
```

Comment:
```suggestion
| **Description** | Concatenate many strings, separated by separator. NULL inputs are skipped. |
```

Reply:
Actually, I didn't read your suggestion thoroughly enough. Concat_ws('-', 'a', NULL, ' b') results in 'a-b', not 'a--b', so the original phrasing was correct. I'm going to make a fixup or for this.

Reply:
Ah, sorry for that.

I thought the differentiation you made by saying "NULL inputs are skipped" for `conncat_ws` but "are treated like empty strings" for `concat` sounded like "NULL inputs in concat_ws are equivalent to just dropping the input" which would incorrectly suggest that `concat_ws('-', NULL)` throws the same error as `concat_ws('-')`. 

Now that I get what you actually meant, I find my interpretation pedantic though and would suggest you just use the wording "NULL inputs are skipped" everywhere (i.e. both `concat_ws` and the string and list versions of `concat`), unless I missed yet another point?

Reply:
Absolutely no worries. That interpretation also makes sense.

Yeah I'll use that everywhere. For arrays, "NULL inputs are skipped" and "NULL inputs are treated as the empty array" are equivalent, but for concat_ws they are not equivalent. since we need the one form for concat_ws, I'll prefer that for arrays too, good idea. PR incoming!

Reply:
ah, you are talking about concat(string, string). They are equivalent there too, so I will also change the phrasing there to be consistent.

Reply:
See https://github.com/duckdb/duckdb-web/pull/4018